### PR TITLE
COR-604 - Use new hospital_nice_choropleth key for hospital choropleths

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**
* Use new key `hospital_nice_choropleth` to display hospital data on Choropleths
* Use `date_unix` from new `hospital_nice_choropleth` key to display in metadata below Choropleths (T-5 days)
* Schema changes

hospital_nice_choropleth

### CODE REVIEW

File: packages/app/schema/gm/__index.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "tested_overall",
     "hospital_nice_sum",
     "hospital_nice",
+    "hospital_nice_choropleth",
```
Comment (VWSCoronaDashboard18) [inline packages/app/schema/gm/__index.json:15]: Does every GM page use this metric or does it only uses the gm_collection

File: packages/app/schema/vr/__index.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "code",
     "tested_overall",
     "hospital_nice",
+    "hospital_nice_choropleth",
```
Comment (VWSCoronaDashboard18) [inline packages/app/schema/vr/__index.json:12]: Does every VR page use this metric or does it only uses the vr_collection

File: packages/app/schema/gm/__index.json (hunk 1 above)
Comment (VWSCoronaDashboard24) [reply packages/app/schema/gm/__index.json:15]: This needs some more investigation and we will pick it up separately. (COR-629)

File: packages/app/schema/vr/__index.json (hunk 2 above)
Comment (VWSCoronaDashboard24) [reply packages/app/schema/vr/__index.json:12]: This needs some more investigation and we will pick it up separately. (COR-629)

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks nice

**Pull request merged**